<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApOne</class>
 <widget class="QLabel" name="ApOne">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>421</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>421</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>421</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="pixmap">
   <pixmap resource="mnemo/mnemo.qrc">:/mnemo/one.svg</pixmap>
  </property>
  <widget class="QCheckBox" name="cb_sirup">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>342</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
min-width: 16px;
min-height: 16px;
max-width: 16px;
max-height: 16px;
}
QCheckBox:checked
{
	image: url(:/pict/pict/lib/lamp_green_16x16.png);
}

QCheckBox:unchecked
{
	image: url(:/pict/pict/lib/lamp_off_16x16.png);
}

QCheckBox:indicator
{
	background: rgba(0, 0, 0, 0);
	height: 16px;
	width: 16px;
}
</string>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_sirup_r">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>334</y>
     <width>20</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
min-width: 20px;
min-height: 32px;
max-width: 20px;
max-height: 32px;
}
QCheckBox:checked
{
	
	
	image: url(:/pict/pict/lib/valve_green_20x32.png);
}

QCheckBox:unchecked
{
	
	image: url(:/pict/pict/lib/valve_off_20x32.png);
}

QCheckBox:indicator
{
	background: rgba(0, 0, 0, 0);
	height: 20px;
	width: 32px;
}
</string>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_voda_r">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>334</y>
     <width>20</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
min-width: 20px;
min-height: 32px;
max-width: 20px;
max-height: 32px;
}
QCheckBox:checked
{
	
	
	image: url(:/pict/pict/lib/valve_green_20x32.png);
}

QCheckBox:unchecked
{
	
	image: url(:/pict/pict/lib/valve_off_20x32.png);
}

QCheckBox:indicator
{
	background: rgba(0, 0, 0, 0);
	height: 20px;
	width: 32px;
}
</string>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_voda">
   <property name="geometry">
    <rect>
     <x>77</x>
     <y>342</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
min-width: 16px;
min-height: 16px;
max-width: 16px;
max-height: 16px;
}
QCheckBox:checked
{
	image: url(:/pict/pict/lib/lamp_green_16x16.png);
}

QCheckBox:unchecked
{
	image: url(:/pict/pict/lib/lamp_off_16x16.png);
}

QCheckBox:indicator
{
	background: rgba(0, 0, 0, 0);
	height: 16px;
	width: 16px;
}
</string>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_bila_r">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>334</y>
     <width>20</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
min-width: 20px;
min-height: 32px;
max-width: 20px;
max-height: 32px;
}
QCheckBox:checked
{
	
	
	image: url(:/pict/pict/lib/valve_green_20x32.png);
}

QCheckBox:unchecked
{
	
	image: url(:/pict/pict/lib/valve_off_20x32.png);
}

QCheckBox:indicator
{
	background: rgba(0, 0, 0, 0);
	height: 20px;
	width: 32px;
}
</string>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_bila">
   <property name="geometry">
    <rect>
     <x>117</x>
     <y>342</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
min-width: 16px;
min-height: 16px;
max-width: 16px;
max-height: 16px;
}
QCheckBox:checked
{
	image: url(:/pict/pict/lib/lamp_green_16x16.png);
}

QCheckBox:unchecked
{
	image: url(:/pict/pict/lib/lamp_off_16x16.png);
}

QCheckBox:indicator
{
	background: rgba(0, 0, 0, 0);
	height: 16px;
	width: 16px;
}
</string>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="c2_vakume_r">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>9</y>
     <width>20</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
min-width: 20px;
min-height: 32px;
max-width: 20px;
max-height: 32px;
}
QCheckBox:checked
{
	
	
	image: url(:/pict/pict/lib/valve_green_20x32.png);
}

QCheckBox:unchecked
{
	
	image: url(:/pict/pict/lib/valve_off_20x32.png);
}

QCheckBox:indicator
{
	background: rgba(0, 0, 0, 0);
	height: 20px;
	width: 32px;
}
</string>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="c1_vakume_r">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>17</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
min-width: 16px;
min-height: 16px;
max-width: 16px;
max-height: 16px;
}
QCheckBox:checked
{
	image: url(:/pict/pict/lib/lamp_green_16x16.png);
}

QCheckBox:unchecked
{
	image: url(:/pict/pict/lib/lamp_off_16x16.png);
}

QCheckBox:indicator
{
	background: rgba(0, 0, 0, 0);
	height: 16px;
	width: 16px;
}
</string>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_para_r">
   <property name="geometry">
    <rect>
     <x>306</x>
     <y>219</y>
     <width>20</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
min-width: 20px;
min-height: 32px;
max-width: 20px;
max-height: 32px;
}
QCheckBox:checked
{
	
	
	image: url(:/pict/pict/lib/valve_green_20x32.png);
}

QCheckBox:unchecked
{
	
	image: url(:/pict/pict/lib/valve_off_20x32.png);
}

QCheckBox:indicator
{
	background: rgba(0, 0, 0, 0);
	height: 20px;
	width: 32px;
}
</string>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_para">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>227</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
min-width: 16px;
min-height: 16px;
max-width: 16px;
max-height: 16px;
}
QCheckBox:checked
{
	image: url(:/pict/pict/lib/lamp_green_16x16.png);
}

QCheckBox:unchecked
{
	image: url(:/pict/pict/lib/lamp_off_16x16.png);
}

QCheckBox:indicator
{
	background: rgba(0, 0, 0, 0);
	height: 16px;
	width: 16px;
}
</string>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lc_Vak">
   <property name="geometry">
    <rect>
     <x>183</x>
     <y>60</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="numDigits">
    <number>3</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lc_Vias">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="numDigits">
    <number>3</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lc_Opr">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="numDigits">
    <number>3</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lc_Lev">
   <property name="geometry">
    <rect>
     <x>182</x>
     <y>109</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="numDigits">
    <number>3</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lc_Temp">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="numDigits">
    <number>3</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QProgressBar" name="pb_Lev">
   <property name="geometry">
    <rect>
     <x>198</x>
     <y>143</y>
     <width>21</width>
     <height>172</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>160</red>
         <green>160</green>
         <blue>160</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>160</red>
         <green>160</green>
         <blue>160</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>160</red>
         <green>160</green>
         <blue>160</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>160</red>
         <green>160</green>
         <blue>160</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="mnemo/mnemo.qrc"/>
 </resources>
 <connections/>
</ui>
